pages/Placeholders.md

```diff
@@ -182,7 +182,7 @@ want to use it as condition in the _conditional component_.
 
  * #### `${time <format>}`
  
-   Displays the current time. Example: `${time HH:mm:ss}`. You can find more information about the date format at <https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html>.
```

Comment:
Why remove the link here?

Reply:
I didn't. I only removed the `<>` since GitHub's markdown is smart enough to recognize URL patterns (Unless the Wiki pages use a completely different handling for this...)

Reply:
I checked it and the links are rendered perfectly fine.

See the PlaceholderAPI Wiki's "Expansion Cloud" page as an example:
![image](https://user-images.githubusercontent.com/11576465/126662597-e58b293a-1485-4822-a3a8-8ca95248f264.png)
![image](https://user-images.githubusercontent.com/11576465/126662630-8517f084-60ed-4015-874e-f376194e9e22.png)